docs/desktop/gnome/decibels.md

```diff
@@ -1,64 +1,66 @@
 ---
-title: Decibels
-author: Christine Belzie
-contributors: Steven Spencer, Ganna Zhyrnova 
+title: Decibels Audio Player
+author: Wale Soyinka
+contributors:
+tags:
+  - gnome
+  - desktop
+  - audio
+  - flatpak
 ---
 
-## Introduction
+# **Decibels: A Simple, Modern Audio Player**
 
```

Comment:
So just change this to ## Introduction: **A Simple, Modern Audio Player** OR ## **A Simple, Modern Audio Player** and "Decibels" will come down as the level 1 heading from the meta.